src/backend/api/Fusion.Resources.Domain/Queries/GetContractPersonnelRequest.cs

```diff
@@ -1,27 +1,56 @@
-﻿using Fusion.Integration.Org;
+﻿using Fusion.AspNetCore.OData;
+using Fusion.Integration.Org;
 using Fusion.Resources.Database;
-using Fusion.Resources.Database.Entities;
 using MediatR;
 using Microsoft.ApplicationInsights;
 using Microsoft.EntityFrameworkCore;
 using System;
-using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Fusion.Resources.Domain.Queries
 {
-
-
     public class GetContractPersonnelRequest : IRequest<QueryPersonnelRequest>
     {
+        private ODataQueryParams query = null!;
+
         public GetContractPersonnelRequest(Guid requestId)
         {
             RequestId = requestId;
         }
 
         public Guid RequestId { get; }
 
+        public ODataQueryParams Query
```

Comment:
Should we make the setter private, to force the use of the method. Then we can contain all the logic regarding odata query in the method, not spread around...